### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by parkaransingh**
Verified merge is complete for the endpoint getReviewsUser integration code and updating of the userprofile model.
Pair Programmers: Alexander Goebel and Madhura Gangal

### CODE REVIEW

File: Civility Optics/Views/SearchView.swift [hunk 1]
```diff
@@ -63,17 +55,19 @@ struct SearchView: View {
   }
 }
 
-struct SearchView_Previews: PreviewProvider {
-  static var previews: some View {
-    NavigationView {
-      SearchView(model: SearchViewModel())
-    }
+
```
Comment (Spellbreeze) [inline Civility Optics/Views/SearchView.swift:58]: Remove commented-out code

**Review by Spellbreeze — APPROVED** [review]

File: Civility Optics/Views/Onboarding/RegisterView.swift [hunk 2]
```diff
@@ -229,8 +229,8 @@ struct RegisterView: View {
...
-struct RegisterView_Previews: PreviewProvider {
-    static var previews: some View {
-        RegisterView()
-    }
-}
+//struct RegisterView_Previews: PreviewProvider {
+//    static var previews: some View {
+//        RegisterView()
+//    }
+//}
```
Comment (madhura112) [inline Civility Optics/Views/Onboarding/RegisterView.swift:236]: Can remove code block

File: Civility Optics/Views/Onboarding/profile.swift [hunk 3]
```diff
@@ -87,15 +87,15 @@ struct profile: View {
...
-            static var previews: some View {
-                profile(model: UserProfileModel(email:""))
-        }
-    }
+//    struct profile_Previews: PreviewProvider {
+//
+//            static var previews: some View {
+//                profile(model: UserProfileModel(email:""))
+//        }
+//    }
```
Comment (madhura112) [inline Civility Optics/Views/Onboarding/profile.swift:98]: Can remove this as well

File: Civility Optics/Views/SearchView.swift [hunk 4]
```diff
@@ -10,21 +10,13 @@ import SwiftUI
...
   var body: some View {
     results.navigationBarTitle("Civility Optics")
           .navigationBarBackButtonHidden(true)
-      
-//      TabView {
-//          SearchView(model: .init()).tabItem {
-//                    Label("Search Places", systemImage: "list.dash")
-//                          }
-//
-//                  }
```
Comment (madhura112) [inline Civility Optics/Views/SearchView.swift:25]: Consider getting rid of this as well

**Review by madhura112 — APPROVED** [review]
Code looks good, need some minor formatting changes

**Pull request merged**